### PR TITLE
[FIX] Add production Lot use date on report

### PR DESCRIPTION
**Pull request by hurrinico**

### CODE REVIEW

File: account_invoice_production_lot/views/company_view.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
```
Comment (SimoRubi) [inline account_invoice_production_lot/views/company_view.xml]: add Copyright

File: account_invoice_production_lot/models/account_invoice.py [hunk 2]
```diff
@@ -40,12 +42,21 @@ def _compute_prod_lots(self):
...
-                    for lot in line.prod_lot_ids
-                ])
+                date = ""
+                for lot in line.prod_lot_ids:
+                    if line.invoice_id.company_id.lot_use_date:
+                        if lot.use_date:
+                            date = datetime.strptime(
+                                lot.use_date,
+                                DEFAULT_SERVER_DATETIME_FORMAT).strftime(date_format)
+                        note += u'<li>S/N' + unicode(
```
Comment (SimoRubi) [inline account_invoice_production_lot/models/account_invoice.py]: Maybe missing a space after `S/N`?

**Review by SimoRubi — APPROVED** [review]
Few corrections in my opinion, not blocking anyway

File: account_invoice_production_lot/models/account_invoice.py [hunk 3]
```diff
@@ -40,12 +42,22 @@ def _compute_prod_lots(self):
...
-                ])
+                date = ""
+                for lot in line.prod_lot_ids:
+                    if line.invoice_id.company_id.lot_use_date:
+                        if lot.use_date:
+                            date = datetime.strptime(
+                                lot.use_date,
+                                DEFAULT_SERVER_DATETIME_FORMAT).strftime(
+                                date_format)
+                        note += u'<li>S/N' + unicode(
```
Comment (eLBati) [inline account_invoice_production_lot/models/account_invoice.py:58]: why ```unicode```?

**Comment by hurrinico** [comment]
Close This one because i have to add another dependencies opening this https://github.com/OCA/account-invoice-reporting/pull/78

**Pull request closed without merging**